dashboard/app/cron.yaml

```diff
@@ -24,7 +24,12 @@ cron:
   schedule: every sunday 00:00
 # Update other coverage numbers every day.
 - url: /cron/batch_coverage?days=true&months=true&steps=10
-  schedule: every 24 hours
+  schedule: every day 00:00
+# Clean up coverage db every week.
+# We're adding data w/o transactions.
```

Comment:
Is there a specific reason to do so? If we did that in transactions, would it be unimportant whether there's a parallel batch_coverage operation?

Reply:
We can't fit single merge data into 1 transaction.
I'm doing a few transactions to put everything into "files" first.
At the end, I update the "merge_history" row. Session column points to the "files" records and the total_rows column explains how many bigquery rows were used to build this aggregation.

Insert/update operations are much more expensive. Every cell or index change is counted as a mutation.
For the delete operations every deleted row + index changes are counted as a mutation.

Reply:
It is quite sad to learn that postgres supports insanely big transactions.
Workarounds are not what I'd like us to be focused on...

Reply:
Will it be less problematic if in `SaveMergeResult()` you first insert into `merge_history` and then into `files`?

Reply:
In case of the failure we'll have inconsistent data. Worst case scenario - no records in "files" with record in "merge_history".
What happens now: in case of no record in "merge_history" recalculation will be triggered next day/week.

Reply:
https://github.com/google/syzkaller/issues/5571 to follow-up.